### PR TITLE
oriented_lattice dependent

### PR DESCRIPTION
**Pull request by abuts**
These are the changes, necessary due to Herbert changes

### CODE REVIEW

File: horace_core/sqw/@rundatah/rundatah.m [hunk 1]
```diff
@@ -157,10 +164,16 @@ function clear_det_cache()
             else
                 proj_mode = 2;
             end
-            % Calculate projections            
+            % Calculate projections
             [u_to_rlu,pix_range,pix] = obj.calc_projections_(obj.detdcn_cache,[],proj_mode);
         end
+        function flds = indepFields(obj)
+            flds = indepFields@rundata(obj);
+            flds = [flds(:)',{'transform_sqw'}];
```
Comment (oerc0122) [inline horace_core/sqw/@rundatah/rundatah.m:172]: Does the orientation of the fields actually matter? why not just
```matlab
flds = [flds, 'transform_sqw'];
```

Comment (abuts) [reply horace_core/sqw/@rundatah/rundatah.m:172]: Interesting. Apparently it looks like it works here. Probably it did not work in a previous Matlab versions

File: horace_core/sqw/@rundatah/rundatah.m [hunk 2]
```diff
@@ -24,6 +24,7 @@ function clear_det_cache()
             % clear cached detectors information and detectors directions
             calc_or_restore_detdcn_([]);
         end
+        %
```
[the remark this answers is not in the document]
Comment (abuts) [reply horace_core/sqw/@rundatah/rundatah.m:27]: I like % as it separate methods when the class contents is collapsed

File: horace_core/sqw/@rundatah/rundatah.m [hunk 3]
```diff
@@ -15,15 +15,17 @@
         % positions
         qpsecs_cache = []
     end
-    
+
     properties(Access=private)
         transform_sqw_f_=[];
     end
+
```
Comment (cmarooney-stfc) [inline horace_core/sqw/@rundatah/rundatah.m:22]: comment for whole file, not specifically this line.

Missing the classVersion method.

**Pull request merged**